java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -0,0 +1,29 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Bitmap Commands" group for standalone and cluster
+ * clients.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/?group=bitmap">Bitmap Commands</a>
```

Comment:
Update to point to Valkey docs

Reply:
Specifically, Valkey does not have groups yet. To keep it consistent we decided to keep it as redis for now as there are some pages missing for valkey. Reference: https://github.com/aws/glide-for-redis/pull/1384#discussion_r1592897061